exchanges/orderbook/depth_test.go

```diff
@@ -712,1689 +504,407 @@ func TestPublish(t *testing.T) {
 func TestIsValid(t *testing.T) {
 	t.Parallel()
 	d := Depth{}
-	if !d.IsValid() {
-		t.Fatalf("received: '%v' but expected: '%v'", d.IsValid(), true)
-	}
-	if err := d.Invalidate(nil); !errors.Is(err, ErrOrderbookInvalid) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, ErrOrderbookInvalid)
-	}
-	if d.IsValid() {
-		t.Fatalf("received: '%v' but expected: '%v'", d.IsValid(), false)
-	}
+	assert.True(t, d.IsValid(), "IsValid should return correct value")
+	err := d.Invalidate(nil)
+	assert.ErrorIs(t, err, ErrOrderbookInvalid, "Invalidate should error correctly")
+	assert.False(t, d.IsValid(), "IsValid should return correct value after Invalidate")
 }
 
-func TestHitTheBidsByNominalSlippage(t *testing.T) {
+func TestMovementMethods(t *testing.T) {
```

Comment:
Please move this test next to the test definitions. I do feel like this increased the complexity and maintenance of testing, but those tests aren't really updated frequently enough for concern. Not requesting you revert it, would be hypocritical of me given exchange_wrapper_standards  😄

Reply:
Done 3ae88d8a